### PR TITLE
input form responsive - Manish 

### PR DESCRIPTION
**Pull request by ManishSheela**
Fix some issues related to input from the responsive page

1. Mobile Device (0 to 666px)

mobile-responsive

2. Tablet Device (666px to 768px)

Large-Screen-768x1031

4. larger Screen Devices (768px up to)
5. 
Large-Screen-768x1031

### CODE REVIEW

File: src/components/Meme.js [hunk 1]
```diff
@@ -13,7 +13,11 @@ export default function Meme() {
 	useEffect(function () {
 		fetch(api_url)
 			.then((data) => data.json())
-			.then((data) => setAllMemes(data.data.memes));
+			.then((data) => setAllMemes(data.data.memes))
+			// Handle the error when something wrong with the code
+			.catch((err) =>
+				document.write("<center> <h3>Engine can't understand this code, it's invalid. Please check code and reload </h3> </center> ")
+			);
```
Comment (dev-lovedeep) [inline src/components/Meme.js:20]: do not work on an issue until assigned. Remove this commit, then only I can merge you PR for original issue

**Comment by dev-lovedeep** [comment]
> chrome toggle device toolbar doesn't show the proper results

a meta tag will fix this problem.

File: src/components/Meme.js (hunk 1 above)
Comment (ManishSheela) [reply src/components/Meme.js]: Accidentally pushed into this branch, sorry for the inconvenience. I have removed the commit.

Comment (ManishSheela) [reply src/components/Meme.js]: Please consider my latest commit.

**Pull request merged**